harness/engine/utils.go

```diff
@@ -452,12 +496,14 @@ func (n *Network) SendTransferTxCmd(senderNodeInd, recvNodeInd uint, amount, fee
 		return nil, err
 	}
 
-	logrus.
-		WithField("s_wallet", senderAddr).
-		WithField("r_wallet", recvAddr).
-		WithField("amount", amount).
-		WithField("fee", fee).
-		Info("Sending transfer")
+	if logrus.GetLevel() == logrus.DebugLevel {
```

Comment:
This conditional seems a bit redundant - if we're not on `Debug` level, the entry will not get written.